File: Ruby/Project Euler #1.rb
num = 0
sum = 0
while num <= 1000
	if num % 5 == 0
		sum = sum + num
	end
	if num % 3 == 0
		sum = sum + num
	end
	if num % 5 == 0 and num % 3 == 0
		sum = sum - num
	end
	num = num + 1
end
puts sum

File: Rust/Project Euler #1.rs
fn main()
{
	let mut sum = 0;
	for x in 3..1000
	{
		if x % 5 == 0
		{
			sum += x;
		}
		else if x % 3 == 0
		{
			sum += x;
		}
	}
	print!("The sum is {}", sum);
}
